### PR TITLE
[docs] mention complete command

### PR DESCRIPTION
**Pull request by finn0**
New user who is setting-up development environment may get confused,
where they should run `./manage.py generatedata` command. So it would be
nice idea to mention complete command, whenever it could be possible.

### CODE REVIEW

**Comment by finn0** [comment]
I've also plans for adding some more clarification in [Install liniting tools](https://kitsune.readthedocs.io/en/latest/hacking_howto.html#install-linting-tools), as I discussed it with leo.

File: docs/hacking_howto.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ First, make sure you have run the "Create some data" step above.
 We include some sample data to get you started. You can install it by
 running this command::
 
-    $ ./manage.py generatedata
+    docker-compose exec web ./manage.py generatedata
```
Comment (akatsoulas) [inline docs/hacking_howto.md:111]: I would suggest to move the whole section `Install Sample Data` above the `Get Search working`. The latter leaves the user into a bash shell into the container and running `docker-compose ...` in there won't work. By moving it up we can avoid this scenario

**Review by akatsoulas — APPROVED** [review]
Thanks for this PR @finn0! r+wc :rocket:

File: docs/hacking_howto.md (hunk 1 above)
Comment (finn0) [reply docs/hacking_howto.md:111]: But in my humble opinion, `Install Sample Data` subsection is more suitable under `Further Setup` section. Not everyone needs to generate sample data, it's an additional step for those who wants to work on Kitsune instance which contains some data. What's your thought now? If still, wants stick to your proposed changes, I'm more than happy to make those changes.

Comment (akatsoulas) [reply docs/hacking_howto.md:111]: Content wise I agree with what you are saying. All I am saying is that the user might get confused by this command b/c the last step left the user inside the container. Another solution here could be to add the `docker-compose exec web` command to the search part too for consistency.

Comment (finn0) [reply docs/hacking_howto.md:111]: Actually, `docker-compose exec web` command already mentioned in [Get search working](https://kitsune.readthedocs.io/en/latest/hacking_howto.html#get-search-working) section, see first instruction. I'm thinking about another solution, adding one another step (4th step) to exit from `web`'s bash shell. Something like,

4. Now exit from web's bash shell: `exit`

and adding `docker-compose exec web` command to other remaining sections, also like [Product Details Initialization](https://kitsune.readthedocs.io/en/latest/hacking_howto.html#product-details-initialization), [Pre-compiling JavaScript Templates](https://kitsune.readthedocs.io/en/latest/hacking_howto.html#pre-compiling-javascript-templates). What do you think? Is it a good idea?

Comment (akatsoulas) [reply docs/hacking_howto.md:111]: sounds good!

**Comment by finn0** [comment]
@akatsoulas, after going through [hacking-howto](https://kitsune.readthedocs.io/en/latest/hacking_howto.html) page, I realized that `$` sign is being used with command which runs in web's bash shell. I tried to maintain this style in this PR, also. But all commands which are present in [Install linting tools](https://kitsune.readthedocs.io/en/latest/hacking_howto.html#install-linting-tools) section have `$` sign with them, despite their are being running outside the web's bash shell. Should I remove `$` sign from all commands present in [Install linting tools](https://kitsune.readthedocs.io/en/latest/hacking_howto.html#install-linting-tools) section?

**Comment by akatsoulas** [comment]
@finn0 I believe it's fine to leave things as is. Thank you for this PR. This should be good to go!

**Comment by finn0** [comment]
@akatsoulas any idea why build is failing?

**Pull request merged**